### PR TITLE
Use variable for mini-repo

### PR DESCRIPTION
**Pull request by m-wynn**
I'd test this in check mode, but my secrets are out of date, and we don't have tags

### CODE REVIEW

File: group_vars/utility.yml [hunk 1]
```diff
@@ -23,3 +23,10 @@ constellation_keyfile_path: /var/lib/acme/live/constellation.collegiumv.org/priv
 
 rsyslog_remote: ""
 rsyslog_server: true
+
+mini-builder-packages:
```
Comment (kulinacs) [inline group_vars/utility.yml]: Ansible doesn't understand variable names with dashes in them

File: roles/xbps-mini-repo/tasks/main.yml [hunk 2]
```diff
@@ -40,14 +40,19 @@
...
-    dest: /opt/xbps-mini-repo/{{ item }}
+    src: xbps-src.conf
+    dest: /opt/xbps-mini-repo/xbps-src.conf
+    owner: root
+    group: root
+    mode: 0644
+
+- name: Template package list
+  template:
+    src: packages.list
```
Comment (kulinacs) [inline roles/xbps-mini-repo/tasks/main.yml]: Could we make this packages.list.j2?

**Review by kulinacs — CHANGES_REQUESTED** [review]
Like the change overall, just some minor fixes

**Comment by m-wynn** [comment]
fixed, probably

**Pull request merged**